tf2onnx/optimizer/transpose_optimizer.py

```diff
@@ -482,8 +482,21 @@ def _mul_handler(self, trans, node):
                 self._g.remove_node(node.name)
                 return True
 
-        # if the shape is () or (1), we just move transpose after the mul
-        if not multiplier.shape or (len(multiplier.shape) == 1 and multiplier.shape[0] == 1):
+        # if the shape is (), we just move transpose after the mul
+        if not multiplier.shape:
+            return self._switch_transpose_and_node(node, trans)
+
+        # if multiplier is 1-D
+        if len(multiplier.shape) == 1:
+            if multiplier.shape[0] == 1:

```

Comment:
Is it possible for "if..." at line 491 be merged to 486 like:
if not multiplier.shape or multipier.shape == [1]:
?

Reply:
Updated unit test to see if 1D const tensors can be jumped over.

For the 'if' merging, it seems cleaner to separate handling of scalars and 1D tensors into 2 blocks (mostly for readability)